.github/workflows/AssignReviewers.yml

```diff
@@ -0,0 +1,31 @@
+## @file
+# Assign reviewers from a REVIEWERS file using CODEOWNERS syntax
+#
+# Copyright (c) 2022, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+# Copied from https://github.com/mdkinney/github-action-assign-reviewers
+# commit 77731e1
+
+name: Assign reviewers from a REVIEWERS file using CODEOWNERS syntax
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - main
+
+jobs:
+  assign_reviewers:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Checkout Pull Request Target
+        uses: actions/checkout@v2
+      - name: Assign Reviewers
+        uses: mdkinney/github-action-assign-reviewers@main

```

Comment:
We might want to consider a tianocore "devops" repository we pipeline templates, actions, reusable workflows, etc. can be centrally located with organization visibility (not essential now just noting).

Reply:
Agreed. This isn't something we'll do now though, so I'll mark this resolved.